infra/helper.py

```diff
@@ -162,6 +174,8 @@ def build_image(build_args):
 def build_fuzzers(build_args):
   """Build fuzzers."""
   parser = argparse.ArgumentParser('helper.py build_fuzzers')
+  _add_engine_args(parser)
+  _add_sanitizer_args(parser)
   parser.add_argument('-e', action='append', help="set environment variable e.g. SANITIZER=address")
```

Comment:
We need to remove the example then, since it is now controlled by --sanitizer argument.